mrc-ide/wodin
# Mrc 3207 Display code errors

This PR displays errors on code tab and run tab when running model endpoints

There are two separate errors that may potentially occur from the endpoints,
* error from odinModelResponse and
* odinRunner

I have added error handlers and display error from odinModelResponse on the left hand side, below validation error while odinRunner error will be displayed on the run section.

please not that I have merged this ticket  https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-3290 into the current Pr. I will subsequently be labelling as duplicate.
To test:

sabotage wodin api by stopping or restarting docker container
docker restart odin.api
quickly type rubbish on code editor
you should see an error displayed.
